lambdas/vacancy-ad/vacancy-ad-mapping.ts

```diff
@@ -0,0 +1,14 @@
+import { VacancyAd } from '../graphql/dto';
+import { toAWSDateTime } from '../graphql/helpers';
+import { VacancyAdRow } from '../database/schema/vacancy-ad-table';
+
+export function toVacancyAdDto({ id, name, description, collaborative, active, createdAt }: VacancyAdRow): VacancyAd {
+  return {
+    id,
+    name,
+    description,
+    collaborative,
+    active,
+    createdAt: toAWSDateTime(createdAt),
```

Comment:
The `categories` field will be `null` even though `schema.graphql` says it shouldn't be.

Reply:
Removed those fields from` schema.graphql` for now. I will add them in `vacancyAd` fetching task.

Reply:
Ok 👍